kafka/kafka-2.1.1/image.yaml

```diff
@@ -1,40 +1,69 @@
 schema_version: 1
 
-name: amq7/amq-streams-kafka
-description: "AMQ Streams image for Kafka"
-version: "1.1.0-kafka-2.1.1"
-from: amq7/amq-streams-kafka-base:1.1.0-kafka-2.1.1
+name: amq7/amq-streams-kafka-2.1
+description: "AMQ Streams image for running Apache Kafka, Zookeeper, Kafka Connect and Mirror Maker"
+version: "1.2.0"
+from: rhel7:7-released
 
 labels:
   - name: "com.redhat.component"
-    value: "amqstreams11-kafka-211-openshift-container"
+    value: "amqstreams12-kafka-21-openshift-container"
   - name: "io.k8s.description"
     value: "A reliable and fault tolerant stream processing platform"
   - name: "io.k8s.display-name"
     value: "AMQ Streams Apache Kafka"
   - name: "io.openshift.tags"
     value: "messaging,amq,jboss"
 
+envs:
+  - name: "KAFKA_HOME"
+    value: "/opt/kafka"
+  - name: "STUNNEL_HOME"
+    value: "/opt/stunnel"
+  - name: "S2I_HOME"
+    value: "/opt/kafka/s2i"
+  - name: "STRIMZI_VERSION"
+    value: "0.12.0"
+  - name: "COM_REDHAT_COMPONENT"
+    value: "amqstreams12-kafkabase-21-openshift-container"
+
 modules:
   repositories:
+    - name: cct_module
+      git:
+        url: https://github.com/jboss-openshift/cct_module.git
+        ref: sprint-28
     - path: ../modules
```

Comment:
what do you think about giving it a name, I have used `modules` for the operator image for example.